packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/message-channel.object-metadata.ts

```diff
@@ -23,12 +23,21 @@ import { MessageThreadObjectMetadata } from 'src/workspace/workspace-sync-metada
 @IsSystem()
 export class MessageChannelObjectMetadata extends BaseObjectMetadata {
   @FieldMetadata({
-    // This will be a type select later: metadata, subject, share_everything
-    type: FieldMetadataType.TEXT,
+    type: FieldMetadataType.SELECT,
     label: 'Visibility',
     description: 'Visibility',
     icon: 'IconEyeglass',
-    defaultValue: { value: 'metadata' },
+    options: [
+      { value: 'default', label: 'Default', position: 0, color: 'green' },
+      { value: 'subject', label: 'Subject', position: 1, color: 'blue' },
+      {
+        value: 'share_everything',
+        label: 'Share Everything',
+        position: 2,
+        color: 'orange',
+      },
+    ],
+    defaultValue: { value: 'default' },
```

Comment:
Replace 'default' with metadata and the default value should be 'share_everything'